components/atom/button/src/index.js

```diff
@@ -63,12 +63,14 @@ const AtomButton = (props) => {
 
   return (
     <Button {...newProps} className={classNames} title={title} disabled={disabled}>
-      {leftIcon && <span className={`${CLASS}-leftIcon`}>{leftIcon}</span>}
-      {leftIcon || rightIcon
-        ? <span className={`${CLASS}-text`}>{children}</span>
-        : children
-      }
-      {rightIcon && <span className={`${CLASS}-rightIcon`}>{rightIcon}</span>}
+      <div className={`${CLASS}-inner`}>
```

Comment:
Div not allowed in button or a as only phrasing content is allowed.

Reply:
ok, we should use an span instead.